cmd/format.go

```diff
@@ -18,6 +18,13 @@ package main
 import (
 	"bytes"
 	"fmt"
+	"github.com/google/uuid"
```

Comment:
What's the formatting rule are you using in your IDE?

Reply:
I forgot to configure `goimports` for this new project on Goland.

Reply:
Fixed.